src/ol/objectutil.js

```diff
@@ -68,3 +68,21 @@ ol.object.isEmpty = function(object) {
   }
   return !property;
 };
+
+
+/**
+ * Adds a `getInstance` static method that always returns the same
+ * instance object.
+ * @param {!Function} object The constructor for the class to add the static
+ *     method to.
+ */
+ol.object.addSingletonGetter = function(object) {
+
+  object.getInstance = function() {
```

Comment:
I'm curious if the Compiler knows anything about the type returned by this function.